internal/datasourcev2/polling_response.go

```diff
@@ -0,0 +1,48 @@
+package datasourcev2
+
+import "github.com/launchdarkly/go-server-sdk/v7/internal/fdv2proto"
+
+// PollingResponse represents the result of a polling request.
+type PollingResponse struct {
+	events   []fdv2proto.Event
+	cached   bool
+	intent   fdv2proto.IntentCode
```

Comment:
I would think we would want to rename this as well. But I could be swayed I suppose. 😄